src/base.iced

```diff
@@ -135,13 +145,17 @@ exports.Base = class Base
       apiUrl = 'https://bitcoinfees.21.co/api/v1/fees/list'
       await request apiUrl, esc defer resp, body
       if resp.statusCode == 200
-        fees = JSON.parse(body)['fees']
+        await a_json_parse body, esc defer body_json
+        await @aget body_json, 'fees', esc defer fees
```

Comment:
well i've never seen this done before, but it works.  i guess if i were doing this i would probably write:

```iced
currentClearanceMins = 10000
for {maxMinutes,MaxFee} in (body_json.fees or []) when maxMinutes <= currentClearanceMins
  currentClearanceMins = maxMinutes
  currentFee = maxFee
  
```